packages/synapse-react-client/src/lib/containers/MarkdownPopover.tsx

```diff
@@ -1,124 +1,105 @@
-import { PositioningStrategy } from '@popperjs/core'
 import React, { useState } from 'react'
-import { Button } from '@mui/material'
-import { Placement } from 'react-bootstrap/esm/Overlay'
-import { ButtonVariant } from 'react-bootstrap/esm/types'
-import { usePopper } from 'react-popper'
-import { CSSTransition } from 'react-transition-group'
+import {
+  Button,
+  Box,
+  TooltipProps,
+  tooltipClasses,
+  ButtonProps,
+} from '@mui/material'
 import MarkdownSynapse, {
   MarkdownSynapseProps,
 } from './markdown/MarkdownSynapse'
 import { Typography } from '@mui/material'
+import LightTooltip from '../components/styled/LightTooltip'
 
 export type MarkdownPopoverProps = {
   children: JSX.Element
   contentProps: MarkdownSynapseProps
-  style?: React.CSSProperties
-  placement?: Placement
+  sx?: TooltipProps['sx']
+  placement?: TooltipProps['placement']
   showCloseButton?: boolean
   actionButton?: {
     content: React.ReactElement
-    variant: ButtonVariant
+    color?: ButtonProps['color']
+    variant?: ButtonProps['variant']
     onClick: () => void
-    closePopoverOnClick: boolean
+    closePopoverOnClick?: boolean
   }
-  /* Useful if the item is in an overflow container. See https://popper.js.org/docs/v2/constructors/#strategy */
-  strategy?: PositioningStrategy
+  maxWidth?: string
+  minWidth?: string
+}
+
+const buttonBoxSx = {
+  display: 'flex',
+  justifyContent: 'space-between',
+  flexDirection: 'row-reverse',
+  '> button': {
+    flexGrow: 1,
+    maxWidth: '45%',
+  },
 }
 
 export const MarkdownPopover: React.FunctionComponent<MarkdownPopoverProps> = ({
   children,
   contentProps,
-  placement,
+  placement = 'bottom-start',
   showCloseButton = true,
   actionButton,
-  strategy,
-  style,
+  sx,
+  maxWidth = '500px',
+  minWidth = '300px',
```

Comment:
Separate general styling from maxWidth/minWidth props, so it easier to change the [tooltip width](https://mui.com/material-ui/react-tooltip/#variable-width) in other components, e.g. HelpPopover